handlers/watchers/watchers.ini

```diff
@@ -1,17 +1,17 @@
 [servo/servo]
-emilio = components/script/dom/webgl components/canvas/webgl ports/geckolib components/style/ components/layout/
-bholley = components/style/ ports/geckolib/ components/selectors/ components/servo_arc/
-Wafflespeanut = python/
-jgraham = components/webdriver_server/ tests/wpt/ -tests/wpt/mozilla -tests/wpt/metadata -tests/wpt/web-platform-tests -tests/wpt/metadata-css -tests/wpt/css-tests
-KiChjang = components/script components/script_traits components/net components/net_traits
-aneeshusa = etc/ci/
-fitzgen = components/script components/script_traits components/profile components/profile_traits components/devtools components/devtools_traits components/debugger
-asajeffrey = components/constellation components/script components/webdriver_server
-edunham = servo-tidy.toml appveyor.yml dependencyci.yml .travis.yml python/tidy
-rillian = rust-stable-version geckolib-rust-toolchain
-canaltinova = components/style/
-paulrouget = components/compositing components/constellation components/servo ports/servo ports/glutin
-cbrewster = components/constellation
+emilio = components/script/dom/webgl/* components/canvas/webgl/* ports/geckolib/* components/style/* components/layout/*
+bholley = components/style/* ports/geckolib/* components/selectors/* components/servo_arc/*
+Wafflespeanut = python/*
+jgraham = components/webdriver_server/* tests/wpt/* -tests/wpt/mozilla/* -tests/wpt/metadata/* -tests/wpt/web-platform-tests/* -tests/wpt/metadata-css/* -tests/wpt/css-tests/*
+KiChjang = components/script/* components/script_traits/* components/net/* components/net_traits/*
+aneeshusa = etc/ci/*
+fitzgen = components/script/* components/script_traits/* components/profile/* components/profile_traits/* components/devtools/* components/devtools_traits/* components/debugger/*
+asajeffrey = components/constellation/* components/script/* components/webdriver_server/*
+edunham = servo-tidy.toml appveyor.yml dependencyci.yml .travis.yml python/tidy/*
+rillian = rust-stable-version/* geckolib-rust-toolchain/*
```

Comment:
`rust-stable-version` and `gecko-rust-toolchain` are actually files, not directories.